### PR TITLE
fs/spiffs: improve documentation

### PR DESCRIPTION
**Pull request by vincent-d**
Slightly improve SPIFFS documentation and change  `SPIFFS_FS_FD_SPACE_SIZE` to better match the upstream requirements.

### CODE REVIEW

File: pkg/spiffs/doc.txt [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /**
  * @defgroup pkg_spiffs   SPI flash file system
  * @ingroup  pkg
- * @ingroup  sys_fs
```
Comment (miri64) [inline pkg/spiffs/doc.txt]: Mhhh. Why remove this line? @aabadie introduced it intentionally in https://github.com/RIOT-OS/RIOT/pull/7298

Comment (miri64) [inline pkg/spiffs/doc.txt]: (also unrelated to the rest of this PR)

Comment (vincent-d) [inline pkg/spiffs/doc.txt]: There is already a spiffs group in `sys_fs` regarding RIOT integration. It looks inconsistent IMO as it appears twice in the  FS group in system.

Comment (miri64) [inline pkg/spiffs/doc.txt]: Then jI would rather add the riot-integration to this `pkg_spiffs`.

Comment (vincent-d) [inline pkg/spiffs/doc.txt]: Indeed, will be better.

Fixed

**Comment by aabadie** [comment]
Please squash

**Comment by vincent-d** [comment]
Squashed

**Comment by aabadie** [comment]
and go

**Pull request merged**